Core1Platform/UserInterface/02_Dashboard.md

```diff
@@ -0,0 +1,211 @@
+
+![Edit tab](../../Assets/Screenshots/Core1Platform/UsingCore1/DashboardEditTab.png "[Edit tab]")
+
+- *Select Viewport*   
+    Click the drop-down list to select the view to be applied to the tab. A different view can be selected for each tab. The drop-down list is only displayed in the *Edit tab* view. The following options are available:    
+    - **Automatic**   
+        The system automatically chooses the dashboard that has a suitable size for the current device.
+    - **Mobile**   
+        The dashboard size is optimized for a mobile device.  
+    - **Tablet**   
+        The dashboard size is optimized for a tablet device.
+    - **Laptop**   
+        The dashboard size is optimized for a laptop device. 
+    - **Desktop**   
+        The dashboard size is optimized for a desktop device.   
+
+
+- [ADD ELEMENT]   
+    Click the button to add a dashlet to the selected tab. The *Dashlet picker* window is displayed.
+
+- [SAVE]   
+    Click this button to save any changes. The changes are applied. The *Edit tab* view is closed.
+
+- ![Cancel](../../Assets/Icons/Cross02.png "[Cancel]") (Cancel)   
+    Click this button to cancel editing the tab. Any changes are rejected. The *Edit tab* view is closed.
+
+- *Dashlet*   
+    Hover over a dashlet until the cursor icon changes to the cross arrows cursor. Then move the dashlet on the predefined grid using drag and drop.    
+    To remove a dashlet from the tab, click the ![Remove](../../Assets/Icons/Cross03.png "[Remove]") (Remove) button in the upper right corner of the dashlet.  
+
+
+### Dashlet picker
+
+*Actindo Dashboard > Select tab > Button Settings > Button ADD ELEMENT*   
+*Actindo Dasboard > Button Add > Button Save > Button Settings > Button ADD ELEMENT*  
+
+![Dashlet picker](../../Assets/Screenshots/Core1Platform/UsingCore1/DashboardDashletPicker.png "[Dashlet picker]")
+
+- [x]    
+    Select a checkbox to add the selected dashlet to the tab. You can select multiple checkboxes. If you click the checkbox in the header, all dashlets in the list are selected. A toolbar indicating the number of selected dashlets is displayed above the list when you have selected at least one checkbox.   
+    By default, the following dashlets are available:  
+
+    > [INFO] For some statistic dashlets, you can influence the data to be evaluated. These dashlets are indicated with an ![Points](../../Assets/Icons/Points02.png "[Points]") (Edit dashlet) button in the upper right corner. Click the button to define the period or the data set to be evaluated.   
+    Additionally, the *Force cash refresh* toggle is available for some dashlets. If you enable the toggle, the data is newly evalatuated. Note that this might take some time.   
+
+    **Actindo Workflow engine**    
+    For detailed information, see [Dashlet picker](../../ActindoWorkFlow/UserInterface/01a_Overview.md#dashlet-picker "Dashlet picker") in the *Actindo Work Flow Engine* documentation.
```

Comment:
Das Modul heiß in Overview *Actindo Work Flow Engine (Workflows)*... den Namen hat sich aber jetzt offiziell geändert, also nur *Workflows*, oder?
https://github.com/Actindo-AG/Documentation/blob/ACD-839/ActindoWorkFlow/Overview/01_General.md